zeroconf/__init__.py

```diff
@@ -2197,7 +2197,11 @@ def remove_all_service_listeners(self) -> None:
             self.remove_service_listener(listener)
 
     def register_service(
-        self, info: ServiceInfo, ttl: Optional[int] = None, allow_name_change: bool = False
+        self,
+        info: ServiceInfo,
+        ttl: Optional[int] = None,
+        allow_name_change: bool = False,
+        cooperating_responders: bool = False,
```

Comment:
Can you document the use case for the new parameter maybe? It'll be helpful to have a few words in the [Sphinx-built documentation](https://python-zeroconf.readthedocs.io/en/latest/api.html#zeroconf.Zeroconf.register_service).

Reply:
Updated PR with a commit as requested

Reply:
Cool, I allowed myself to add few more words there.